src/Nethermind/Nethermind.Blockchain.Test/BeaconBlockRootHandlerTests.cs

```diff
@@ -149,6 +149,6 @@ public void Test_StoreBeaconRoot_AccessListNotNull()
 
         transaction.Hash = transaction.CalculateHash();
         _transactionProcessor.Received().Execute(Arg.Is<Transaction>(t =>
-            t.Hash == transaction.Hash), header, NullTxTracer.Instance);
+            t.Hash == transaction.Hash), new BlockExecutionContext(header, 0), NullTxTracer.Instance);
```

Comment:
this is assuming that system transaction will not have blobs, so 0 blobBaseFee will not be a issues.